src/common/com/intellij/plugins/haxe/util/HaxeStringUtil.java

```diff
@@ -228,4 +228,21 @@ public static String elide(@Nullable String s, int len) {
     return s.substring(0, len - ELLIPSES.length()) + ELLIPSES;
   }
 
+
+  /**
+   * Gets a version of the given string with a lower-cased first character.
+   * @param s - string to change.
+   */
+  @Nullable
+  public static String toLowerFirst(@Nullable String s) {
```

Comment:
Could be replaced with:
`com.intellij.openapi.util.text.StringUtil#decapitalize`

Reply:
No, because that refuses to de-capitalize when the second letter is also a capital (as in URL).